src/utils/checkDarkMode.tsx

```diff
@@ -0,0 +1,5 @@
+import {useColorScheme} from 'react-native';
+
+export default function isLightMode(): boolean {
+  return useColorScheme() === 'dark';
```

Comment:
Is this logic correct? It seems like it's flipped from what it should be.

Reply:
This was an old function title. Its been renamed to match the current use of the function.